webui/react/src/pages/F_ExpList/F_ExperimentList.settings.ts

```diff
@@ -104,7 +104,7 @@ export const settingsConfigForProject = (id: number): SettingsConfig<F_Experimen
       type: string,
     },
   },
-  storagePath: `f_project-details-${id}`,
+  storagePath: `experimentListingForProject${id}`,
```

Comment:
We're mostly using kebab case for these but since they're know visible in the settings drawer and we're not 100% consistent how do people feel about using camelcase for consistency with everything else?